service/src/main/java/com/epam/esm/dto/CertificateDTO.java

```diff
@@ -0,0 +1,228 @@
+package com.epam.esm.dto;
+
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Certificate transfer class
+ */
+public class CertificateDTO {
+
+    private BigInteger id;
+
+    private String name;
+
+    private String description;
+
+    private BigDecimal price;
+
+    private int duration;
+
+    private LocalDateTime createDate;
+
+    private LocalDateTime lastUpdateDate;
+
+    private List<TagDTO> tagsDTO;
```

Comment:
A certificate can only contain unique tags.
Maybe you should then use Set instead of List?